lib/assets/javascripts/cable.coffee

```diff
@@ -0,0 +1,19 @@
+#= require_self
+#= require cable/consumer
+
+@Cable =
+  PING_IDENTIFIER: "_ping"
+
+  createConsumer: (url) ->
+    new Cable.Consumer websocketUrl(url)
+
+  websocketUrl = (url) ->
+    if url and not /^wss?:/i.test(url)
+      a =  document.createElement('a')
```

Comment:
:scissors: extra space after `=`, and use double quotes (`"a"`) for consistency please.